ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-isis.j2

```diff
@@ -17,15 +17,13 @@ router isis {{ router_isis.instance }}
 {%                 if redistribute_route.include_leaked is arista.avd.defined(true) %}
 {%                     set redistribute_route_cli = redistribute_route_cli ~ " include leaked" %}
 {%                 endif %}
-{%                 if redistribute_route.ospf_route_type is not arista.avd.defined %}
-{%                     continue %}
+{%                 if redistribute_route.ospf_route_type is arista.avd.defined %}
```

Comment:
Given that based on your last comment to @tgodaA if the `ospf_route_type` is not defined it should not be rendered I am thinking that you could leverage the error functionality of the defined plugin in this scenario:
```suggestion
{%                 if redistribute_route.ospf_route_type is arista.avd.defined(fail_action='error') %}
```

Cf https://avd.sh/en/latest/plugins/index.html#defined-test

This way if the ospf_route_type is missing it will raise an an error.

Reply:
I would prefer to just ignore the section if the required var is missing. That is what we do everywhere else in eos_cli_config_gen.
We will add input validation soon, so let's not add it deeply inside some template too.

Reply:
ack - resolving this